packages/babel-make-styles/README.md

```diff
@@ -92,3 +92,31 @@ const useStyles = __styles({
   },
 });
 ```
+
+## Troubleshooting
+
+This section focuses mainly on troubleshooting this babel plugin in the [microsoft/fluenui repo](https://github.com/microsoft/fluentui).
+However the concepts are not coupled to the repo setup.
+
+### Linaria
+
+The plugin uses tools from [linaria](https://github.com/callstack/linaria) to evaluate runtime calls of `makeStyles`.
+[Linaria's debugging documentation can help here](https://github.com/callstack/linaria/blob/master/CONTRIBUTING.md#debugging-and-deep-dive-into-babel-plugin). Here are a few examples with building packages with linaria debug output.
+
+Directly from the package
+
+> DEBUG=linaria\* LINARIA_LOG=debug yarn build
+
+Using Lage from the root of the repo
+
+> DEBUG=linaria\* LINARIA_LOG=debug yarn lage build --to <package-name>
+
+Using yarn workspace from the root of the repo
+
+> DEBUG=linaria\* LINARIA_LOG=debug yarn workspace <package-name> build
+
```

Comment:
````suggestion
On Windows it's required to set environment variables via [`set`](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1) or you can use `cross-env`, for example:

```sh
$ yarn cross-env DEBUG=linaria\* LINARIA_LOG=debug yarn workspace <package-name> build
```
````

Usage on Windows it's not the most obvious thing, let's mention it